src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -408,32 +372,13 @@ static BoundInterpolatedString createInterpolation(BoundExpression expression, I
         {
             Debug.Assert(binaryOperator.IsUnconvertedInterpolatedStringAddition);
 
-            var stack = ArrayBuilder<BoundBinaryOperator>.GetInstance();
             var partsArrayBuilder = ArrayBuilder<ImmutableArray<BoundExpression>>.GetInstance();
 
-            BoundBinaryOperator? current = binaryOperator;
-
-            while (current != null)
+            binaryOperator.VisitBinaryOperatorInterpolatedString(partsArrayBuilder, (BoundUnconvertedInterpolatedString unconvertedInterpolatedString, ArrayBuilder<ImmutableArray<BoundExpression>> partsArrayBuilder) =>
```

Comment:
>, [](http://example.com/codeflow?start=82&length=1)

static for the `visitor` lambda? (comment applies to other callers of `VisitBinaryOperatorInterpolatedString` too)